xds/internal/client/bootstrap/bootstrap.go

```diff
@@ -96,6 +96,29 @@ type xdsServer struct {
 	ServerFeatures []string       `json:"server_features"`
 }
 
+func bootstrapConfigFromEnvVariable() ([]byte, error) {
+	fName := env.BootstrapFileName
+	fContent := env.BootstrapFileContent
+
+	// Bootstrap file name has higher priority than bootstrap content.
+	if fName != "" {
+		// If file name is set
+		// - If file not found (or other errors), fail
+		// - Otherwise, use the content.
+		//
+		// Note that even if the content is invalid, we don't failover to the
+		// file content env variable.
+		logger.Debugf("xds: using bootstrap file with name %q", fName)
+		return bootstrapFileReadFunc(fName)
+	}
+
+	if fContent != "" {
+		return []byte(fContent), nil
+	}
+
+	return nil, fmt.Errorf("none of the bootstrap environment variables (%q or %q) defined", "GRPC_XDS_BOOTSTRAP", "GRPC_XDS_BOOTSTRAP_CONFIG")
```

Comment:
Should we export consts for those strings and reference them here?

Reply:
We do have consts for them, but they are in another package.
Exported those and used them here.